apps/publisher/src/app.tsx

```diff
@@ -1,56 +1,59 @@
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import {
```

Comment:
This file is a little long. Ideally, I would like to see this broken up. Because a lot of the same states/variables are used throughout its entirety, I think it would be a good idea to explore wrapping the app in a shared `Context` if we were to break it up.